src/App.js

```diff
@@ -23,6 +24,8 @@ function App() {
       <header className="App">NC Games</header>
       <Nav/>
       <Routes>
+        <Route path='/reviews' element={<Reviews isLoading={isLoading} reviews={reviews} setReviews={setReviews} setIsLoading={setIsLoading}/>}></Route>
```

Comment:
in terms of my isLoading comments, generally we'll aim to keep App.js as clear of state etc as possible, to serve more like a route map of the app